lib/Frenetic_NetKAT_Virtual_Compiler.mli

```diff
@@ -1,15 +1,18 @@
 open Core.Std
 open Frenetic_NetKAT
+open Frenetic_NetKAT_FabricGen
 
-val compile : ?log:bool
-           -> ?record_paths:string
-           -> vrel:pred
-           -> vtopo:policy
-           -> ving_pol:policy
-           -> ving:pred
-           -> veg:pred
-           -> ptopo:policy
-           -> ping:pred
-           -> peg:pred
-           -> policy (* vpol, the virtual policy *)
-           -> policy
+module Make(FG:FABRIC_GEN) : sig
+  val compile : ?log:bool
+    -> ?record_paths:string
+    -> vrel:pred
+    -> vtopo:policy
+    -> ving_pol:policy
+    -> ving:pred
+    -> veg:pred
+    -> ptopo:policy
+    -> ping:pred
+    -> peg:pred
+    -> policy (* vpol, the virtual policy *)
+    -> policy
+end
```

Comment:
I just thought it would make the interface cleaner, because all relevant function are then documented in once place.